src/Pokedex.jsx

```diff
@@ -1,59 +1,27 @@
 import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
 import {
   AppBar,
   Toolbar,
   Grid,
   Card,
   CardMedia,
   CardContent,
-  CircularProgress,
   Typography,
   TextField,
 } from '@material-ui/core';
 import Button from '@material-ui/core/Button';
 import SearchIcon from '@material-ui/icons/Search';
-import { fade, makeStyles } from '@material-ui/core/styles';
-import { toFirstCharUppercase } from './constants';
+import { makeStyles } from '@material-ui/core/styles';
+import { toFirstCharUppercase } from './utils/constants';
 import axios from 'axios';
 
-const useStyles = makeStyles((theme) => ({
-  pokedexContainer: {
-    paddingTop: '20px',
-    paddingLeft: '50px',
-    paddingRight: '50px',
-  },
-  cardMedia: {
-    margin: 'auto',
-  },
-  cardContent: {
-    textAlign: 'center',
-  },
-  searchContainer: {
-    display: 'flex',
-    backgroundColor: fade(theme.palette.common.white, 0.15),
-    paddingLeft: '20px',
-    paddingRight: '20px',
-    marginTop: '5px',
-    marginBottom: '5px',
-  },
-  searchIcon: {
-    alignSelf: 'flex-end',
-    marginBottom: '5px',
-  },
-  searchInput: {
-    width: '200px',
-    margin: '5px',
-  },
-  genButton: {
-    marginLeft: '10px',
-    marginRight: '10px',
-    paddingTop: '10px',
-    paddingBottom: '10px',
-  },
-}));
+// Styles
+import { PokedexStyles } from './styles/PokedexStyles';
 
-const Pokedex = (props) => {
-  const { history } = props;
+const Pokedex = () => {
+  const history = useHistory();
```

Comment:
This removes the need to pass in props. With React hooks, this is the "new" way.